hack/util.sh

```diff
@@ -220,7 +220,7 @@ function tryuntil {
 	timeout=$(($(date +%s) + 90))
 	echo "++ Retrying until success or timeout: ${@}"
 	while [ 1 ]; do
-		if eval "${@}" 2>&1 >/dev/null; then
+		if eval "${@}" >/dev/null 2>&1; then
```

Comment:
me too but i think that's considered deprecated syntax: http://wiki.bash-hackers.org/scripting/obsolete